Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM golang:1.13
+# We need to have both nodejs and go to build the binaries.
+# We could use multi-stage builds but that would require significantly changing our Makefile.
+RUN apt-get update
+RUN curl -sL https://deb.nodesource.com/setup_10.x | bash -
+RUN apt-get update && apt-get install -y nodejs
+
+WORKDIR /aws-amazon-ecs-cli-v2
+COPY . .
+RUN go env -w GOPROXY=direct
```

Comment:
I had to put this because of our network limitations, but customers shouldn't have to.
If I remove it I don't have a way of testing the image :|